geoportal/demo_geoportal/templates/story-map/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
```

Comment:
Should be in: geoportal/demo_geoportal/static/story-map.html

Reply:
done